.github/workflows/manual-post-release.yml

```diff
@@ -18,6 +18,11 @@ on:
         required: true
         type: boolean
         default: false
+      mark_as_latest:
+        description: 'Mark the packages as latest'
+        required: true
+        type: boolean
+        default: true
```

Comment:
we don't want to mark the packages as latest when doing a `patch` when we are a couple of versions behind the latest version